bitex/formatters/bitfinex.py

```diff
@@ -29,32 +32,100 @@ def ticker(self):
 
     def order_book(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json()
+        asks = []
+        bids = []
+        if 'bids' in data:  # api version 1
+            for i in data['asks']:
+                asks.append([i['price'], i['amount']])
```

Comment:
`i` usually represents the index of a list. In this case, however, you're looping over the items of a list of dictionaries.
Better would be to name it `mapping` or `dictionary` or, even better, `quote` or `order`.